.github/workflows/release-CI.yml

```
@@ -202,3 +202,38 @@ jobs:
         run: |
           uv pip install --upgrade twine
           twine upload --skip-existing *.whl
+
+  discord-notify:
+    name: Discord Release Notification
+    runs-on: ubuntu-latest
+    needs: [release]
+    if: always() && needs.release.result == 'success'
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)